net/stun/stun.go

```diff
@@ -225,40 +226,48 @@ func ParseResponse(b []byte) (tID TxID, addr []byte, port uint16, err error) {
 				return err
 			}
 			if len(a) == 16 {
-				addr6, port6 = a, p
+				var ip6 [16]byte
+				copy(ip6[:], a)
```

Comment:
no need for these explicit copies as of Go 1.17: https://go.dev/doc/go1.17#language

If you want you can write:

```
     netip.AddrFrom16(*(*[16]byte)([]byte(a)))
```

But it'll be about identical performance.

Reply:
Ah, I was looking for this syntax. Updated!